core/AssetManager/UIAssetFetcher/PluginUmdAssetFetcher.php

```diff
@@ -34,6 +34,10 @@ class PluginUmdAssetFetcher extends UIAssetFetcher
 
     public function __construct($plugins, $theme, $chunk, $loadIndividually = null, $chunkCount = null)
     {
+        // Login must always be loaded, even if there's another login plugin being used
+        $plugins[] = 'Login';
+        $plugins = array_unique($plugins);
```

Comment:
@diosmosis This actually seem to cause problems in the tests. The AssetMerger tries to post an event. But as the Login plugin isn't loaded, but provided anyway, it fails to get the registered events for this plugin.
Not sure if we should/could simply load the plugin to avoid that issue...

Reply:
Added a plugin mock instance for Login, fixes the test locally for me

Reply:
That fixes the tests, but Matomo doesn't work when you disable the "Login" plugin and use another plugin like LoginLdap instead.

Reply:
Oh I see... I'll look into it.

Reply:
@sgiehl I merged 5.x-dev into this branch, then ran `./console plugin:activate LoginLdap` and everything seems to work. Can you check? Maybe I have to trigger a failure somehow.

Reply:
did you try with merged assets and disabled developer mode?

Reply:
Yes I tried with both. Specifically did a cache clear with merged assets too.

Reply:
Tried again and managed to reproduce, looking into it now

Reply:
@sgiehl pushed something that fixes it for me locally

Reply:
it's fixed now 👌